### DISCUSSION TITLE
toSQL() or fromSQL() not compliant with DATETIME

### DISCUSSION THREAD

**Discussion by femto-code** [moment/luxon#1309] [Q&A] [answered]
As stated here (https://dev.mysql.com/doc/refman/8.0/en/datetime.html) the specification for SQL requires the format (YYYY-MM-DD hh:mm:ss) to comply with the DATETIME data type.

However, luxons SQL-related functions don't seem to produce that format, is that correct? Should I go with toFormat then?

**Comment by icambron** [comment, marked as answer]
`DateTime.now().toSQL({ includeOffset: false } )` should be what you want. The issue is only that MySQL doesn't accept the offset

**Reply by femto-code** [reply]
Alright, I see. But with the option set, it still shows milliseconds.

Is there a way to suppress that like in toISO or do I have to cut it manually?

**Reply by icambron** [reply]
Yeah, it doesn't currently support the `suppressMilliseconds` argument that `toISO()` does. My MySQL does accept milliseconds, IIRC. You can also round down to get 0s there, by calling `startOf("second")` on it first

**Reply by femto-code** [reply]
Okay, I agree. You can have them but I prefer cutting them off then.

Would be nice to see the `suppressMilliseconds` method for the toSQL function call as well though.
